pytroll_collectors/tests/test_region_collector.py

```diff
@@ -100,6 +125,72 @@ def test_collect_duration(europe):
     alt_europe_collector(granule_metadata)
 
 
+@unittest.mock.patch("pyorbital.tlefile.urlopen", new=_fakeopen)
+def test_collect_check_schedules(europe_collector_schedule_cut, caplog):
+    """Test default schedule cut method."""
+    granule_metadata = {
+            "platform_name": "Metop-C",
+            "sensor": "avhrr"}
+
+    with caplog.at_level(logging.DEBUG):
+        for s_min in (0, 3, 6, 9, 12, 15, 18):
+            europe_collector_schedule_cut.collect(
+                    {**granule_metadata,
+                     **{"start_time": datetime.datetime(2021, 4, 11, 10, s_min, 0),
+                        "end_time": datetime.datetime(2021, 4, 11, 10, s_min+3, 0),
+                        "uri": f"file://{s_min:d}"}})
```

Comment:
these lines are duplicated, could be defined outside the functions

Reply:
Hm, the start_time etc have a variable inside, s_min. I must make a function then passing the s_min return the dict.

Reply:
ah, indeed, I missed that. But yes, a function would be nice

Reply:
Something like this?

Reply:
yes, looks good!